cmd/extract-block-events/main.go

```diff
@@ -100,7 +100,7 @@ func run() int {
 		heights[i], heights[j] = heights[j], heights[i]
 	})
 
-	// Go through heights, try to get the block on each height until we reach
+	// Go through heights, try to get the block on each height until reaching
```

Comment:
I know you dislike the comment style where you talk from the perspective of a component, but with this change, it just sounds bad.